stories/topics/proc-gcp-application-upgrade.adoc

```diff
@@ -0,0 +1,146 @@
+.. Update the data file
+.. Run the `ansible-on-clouds-ops` 2.3 container to upgrade your {PlatformNameShort} from {GCP Marketplace} deployment.
+
+. (Optional) Restore the stack from backup.
+.. Pull the ansible-on-clouds-ops 2.2 container image
+.. Prepare the environment
+.. Run the `ansible-on-clouds-ops` 2.2 container to restore the stack
+
+= Backing up your {PlatformNameShort} stack
+
+.Procedure
+
+. Create the directories used to configure the upgrade process using the following commands:
++
+[source,bash]
+----
+# Secrets directory. Store the GCP credentials file.
+$ mkdir secrets
+# Extra vars directory. All the configuration variables for the upgrade process.
+$ mkdir extra_vars
+----
++
+. Store your service account credential JSON file in the secrets directory. 
++
+[NOTE]
+====
+To create the GCP credentials file, see, 
+link:https://developers.google.com/workspace/guides/create-credentials[Create credentials] and 
+link:https://cloud.google.com/iam/docs/keys-create-delete[Create and delete service account keys]
+====
++
+. Gather the following input variables to create an `extra_vars.yaml` file within `extra_vars` dir.
+* *gcp_service_account_credentials_json_path*:  Service account credential json file path for the Google cloud account.  This is the path within an `ansible-on-clouds-ops` container where the credentials JSON file is mounted to the `/secrets` directory. The path should be `/secrets/<service-account-creds-json-filename>`. 
++
+Replace `<service-account-creds-json-filename>` with your Google service account credential filename.
+
+* *gcp_deployment_name*: Deployment name that you want to upgrade.
+* *gcp_compute_region*: GCP region that you provided when deploying foundation deployment. 
+If you have not provided a region when deploying the foundation, use the default region `us-east1` here.
+* *gcp_compute_zone*: GCP zone that you provided when deploying foundation deployment. 
+If you have not provided a zone when deploying the foundation, use the default `us-east1-b` here.
++
+[NOTE]
+====
+You can find the region and zone from the filestore instance deployed with the foundation deployment. 
+Look for the filestore with the name `<deployment-name>-filestore`.
+
+* Make note of the location field that is your `gcp_compute_zone`. 
+For example, the filestore instance location might be `us-east1-d`
+
+* Remove the last two characters from the location that is your `gcp_compute_region`. 
+For example, if your `gcp_compute_zone` location is `us-east1-d`, your region is `us-east1`.
+====
++
+[source,bash]
+----
+gcp_service_account_credentials_json_path: "/secrets/<service-account-creds-json-filename>"
+gcp_deployment_name: "" 
+gcp_compute_region: ""
+gcp_compute_zone: ""
+----
++
+. The final result should look similar to the following:
++
+[source,bash]
+----
+$ tree
+tree
+.
+├── extra_vars
+│   └── extra_vars.yaml
+└── secrets
+    └── gc-ansible-cloud-123434.json
+----
+<<<<<<< HEAD
```

Comment:
Another conflict

Reply:
These conflicts look like they're part of an older PR.